.travis.yml

```diff
@@ -2,6 +2,19 @@
 sudo: required
 
 language: python
+python:
+<<<<<<< HEAD
```

Comment:
It seems that you tried to merge, but the merge commit markers are still included.